src/subscript/runrms/runrms.py

```diff
@@ -81,6 +81,22 @@ def get_parser():
     # positional:
     prs.add_argument("project", type=str, nargs="?", help="RMS project name")
 
+    prs.add_argument(
+        "--project",
+        "-project",
+        dest="project_alt",
+        help="RMS project name (alternative version)",
```

Comment:
Is "alternative version" sufficient for explaining users of `runrms --help` that there are two ways of supplying the project name, or should it be explained more?

Reply:
The idea was full backwards user experience with current `rms -project foo`. I will explain more

Reply:
Detailed explanation added